src/app/api/tasks/tasks.service.ts

```diff
@@ -275,12 +275,10 @@ export class TasksService extends BaseService {
     }
     const labels = tasks.map((task) => task.label)
 
-    await this.db.$transaction([
-      this.db.task.deleteMany({
-        where: { assigneeId, assigneeType, workspaceId: this.user.workspaceId },
-      }),
-      this.db.label.deleteMany({ where: { label: { in: labels } } }),
-    ])
+    await this.db.task.deleteMany({
```

Comment:
any reason for removing transaction?

Reply:
It was throwing an obscure error (only at times)... I couldn't find a solution for it so after multiple tries, the transaction has been removed.

Removing it altogether isn't the best way to approach this, I will look into this in the future.